contracts/common/misc/Pausable.sol

```diff
@@ -0,0 +1,18 @@
+pragma solidity ^0.5.2;
+
+import {Governable} from "../governance/Governable.sol";
+
+contract Pausable is Governable {
```

Comment:
instead of this use `Lockable` `common/mixin/Lockable.sol`

Reply:
Done.